Firmware/ControladorRobo/irarray.cpp

```diff
@@ -70,14 +72,57 @@ void IRArray::readSensors()
     for (int i = 0; i < NUMBER_OF_SENSORS; i++)
     {
         m_sensorRaw[i] = analogRead(m_sensorPin[i]);
-        byte sensorRawByte = m_sensorRaw[i]>>2;
-        if (m_calibrating)
```

Comment:
Maybe we can have two variables to active this method.
m_calibrating to calibrate routine and other like m_continousCalibrate to update values every single time